cli/src/config/templates.toml

```diff
@@ -40,7 +40,7 @@ if(root,
           separate(" ",
             format_short_change_id(change_id) ++ if(divergent, "??"),
             if(hidden, "hidden"))),
-        author.username(),
+        if(author.email(), author.username(), email_placeholder),
```

Comment:
Should the two "email" here be "username"? Or what's the thinking behind it otherwise?

Reply:
At least I read it as email is the one you actually set and username is just a derivative of that (so I think it makes sense the messaging be around setting email and not username).

Reply:
That makes sense. I just confused "username" with "name"